appinfo/routes.php

```diff
@@ -43,6 +43,7 @@
 		['name' => 'card#read', 'url' => '/cards/{cardId}', 'verb' => 'GET'],
 		['name' => 'card#create', 'url' => '/cards', 'verb' => 'POST'],
 		['name' => 'card#update', 'url' => '/cards/{cardId}', 'verb' => 'PUT'],
+		['name' => 'card#clone', 'url' => '/cards/{cardId}/clone', 'verb' => 'POST'],
```

Comment:
Deck currently has this API duplication where we tend to have two endpoints, one for web another one for external API consumers like mobile apps.

Ideally we would introduce new endpoints as OCS API endpoints, that way they can be used by both.

Not a blocker for this merge, but we should follow up that part before the final release.